shumway/__init__.py

```diff
@@ -92,13 +92,21 @@ def __exit__(self, *args):
 
 class MetricRelay(object):
     """Create and send metrics"""
-    def __init__(self, default_key, ffwd_host=FFWD_IP, ffwd_port=FFWD_PORT,
-                 ffwd_path=None, default_attributes=None, use_http=False):
+    def __init__(self, default_key, ffwd_host=None, ffwd_ip=None,
+                 ffwd_port=FFWD_PORT, ffwd_path=None, default_attributes=None,
+                 use_http=False):
+        if ffwd_host is not None:
+            host = ffwd_host
+        elif ffwd_ip is not None:
+            host = ffwd_ip
```

Comment:
If someone passes both of these in I think an error should be raised. It doesn't make sense to specify both host and IP.

Reply:
yep makes sense to throw an error in that case - I changed it accordingly